package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "eslint src",
     "lint:fix": "eslint src --fix",
     "prettier": "prettier --check .",
-    "prettier:fix": "prettier --write ."
+    "prettier:fix": "prettier --write .",
+    "test": "jest"
```

Comment:
I would advise you to separate test view modes for speed

```json
    "test": "jest --watch",
    "test:coverage": "jest --coverage",
```

in watch mode, tests will be run after each change
in coverage mode, test coverage will be additionally displayed